BUILD.bazel

```diff
@@ -1050,9 +1196,12 @@ kernel_build(
     ],
     build_config = "build.config.rpi4",
     make_goals = [
+         "modules",
          "Image.gz",
          "dtbs",
     ],
+    module_outs = _RPI4_MODULE_OUTS,
+    strip_modules = True,
```

Comment:
@gajavrbinc what does this do?

Reply:
This line basically stripps away unnecessary symbols. It decreases size. False is mostly only used for debbuging.